apps/src/applab/applab.js

```diff
@@ -8,14 +8,14 @@
 'use strict';
 
 var studioApp = require('../StudioApp').singleton;
-var commonMsg = require('../../locale/current/common');
-var applabMsg = require('../../locale/current/applab');
+var commonMsg = window.blockly.locale;
+var applabMsg = window.blockly.applab_locale;
```

Comment:
yeah, i agree. it should be possible for me to convert these back to (non-relative) require calls with a bit more work. ideally the end result would be something like `var applabMsg = require('@cdo/applab_locale')` or maybe `require('@cdo/locale/applab')`.